packages/block-library/src/categories/edit.js

```diff
@@ -180,72 +186,159 @@ export default function CategoriesEdit( {
 	const blockProps = useBlockProps( {
 		className: classes,
 	} );
+	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
 
 	return (
 		<TagName { ...blockProps }>
 			<InspectorControls>
-				<PanelBody title={ __( 'Settings' ) }>
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							displayAsDropdown: false,
+							showHierarchy: false,
+							showPostCounts: false,
+							showOnlyTopLevel: false,
+							showEmpty: false,
+							showLabel: true,
+						} );
+					} }
+					dropdownMenuProps={ dropdownMenuProps }
+				>
 					{ Array.isArray( taxonomies ) && (
-						<SelectControl
-							__nextHasNoMarginBottom
-							__next40pxDefaultSize
+						<ToolsPanelItem
+							hasValue={ () => {
+								return taxonomySlug !== 'category'
+									? taxonomySlug !== 'post_tag'
+									: taxonomySlug !== 'category';
+							} }
```

Comment:
```suggestion
							hasValue={ () => taxonomySlug !== 'category' }
```

Is there any reason to use a complex conditional statement? In this case, it seems like it would be fine to just check for the default value `category`.